docs/yaml/elements/precinct.yaml

```diff
@@ -31,9 +31,7 @@ tags:
   description: Links to an :ref:`$$$-electoral-district` (e.g., congressional district,
     state house district, school board district) to which the precinct belongs. **Highly
     Recommended** if candidate information is to be provided. Multiple allowed and
-    recommended to specify the geography of multiple electoral districts. If an electoral
-    district splits a precinct, use the `PrecinctSplitName` object and do not specify
-    that particular electoral district in this object.
+    recommended to specify the geography of multiple electoral districts.
```

Comment:
Yes, I agree.  Reviewing this now, I think the documentation should note that the electoral districts should be _wholly contained_.  Suggest: changing  "to which the precinct belongs" to be "to which the entire precinct or precinct split is contained within).

Without some note like that, I fear it could be interpreted as any intersection with the precinct/spit.